docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "2"
+services:
+  apostrophe:
+    container_name: apostrophe
+    restart: always
+    build: .
+    ports:
+      - "3000:3000"
+    links:
+      - mongo
+    volumes:
+      - ./data/uploads:/app/public/uploads
```

Comment:
Wonder if we should add a note in the readme to also add the lib folder if you're using this for development. Otherwise, you'd have to restart the container on every change.